### ISSUE TITLE
AILayer - AINN2

### ISSUE THREAD

**Issue by yuxlantian** [NDLABS-Leo/Allocator-Pathway-ND-CLOUD#47]
### Version

1

### DataCap Applicant

AINN Layer2

### Project ID

007 

### Data Owner Name

AILayer

### Data Owner Country/Region

United States

### Data Owner Industry

Web3 / Crypto

### Website

https://anvm.io/

### Social Media Handle

https://x.com/AILayerXYZ

### Social Media Type

Twitter

### What is your role related to the dataset

Dataset Owner

### Total amount of DataCap being requested

6PiB

### Expected size of single dataset (one copy)

600TiB

### Number of replicas to store

10

### Weekly allocation of DataCap requested

1PiB

### On-chain address for first allocation

f1ix2cf2d5pmdiw3of7eou34ftu4btklqng7eo45q

### Data Type of Application

Private Commercial/Enterprise

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
AINN Layer2 Description： 
 
- Narrative Innovation: The first Bitcoin Layer 2 aimed at supporting the mass adoption of AI applications, with the goal of advancing the deep integration of Web3 and AI, benchmarking Merlin & Bittensor. 
 
- Community Support: An ecosystem project strongly supported by the AINN community, currently one of the first key supported inscription by the Binance web3 wallet, with a market value exceeding $50 million. AINN Layer2 will use AINN as gas consumption and the issued assets as governance tokens, with AINN's strong community and ecological support. 
 
- Technology Leadership: Bitcoin EVM-compatible Layer2 with multiple AI subnets to lay a foundation for bridging AI models, data, and algorithms. 
 
- Strong Background: The team consists of graduates from top American universities, chief scientists from startup financed by top venture capital such as Sequoia, and also get support from some of the largest Bitcoin holders. The initial phase targets a TVL of $1 billion. 
 

- Project Stage: Currently in the social point fission stage, with over 70,000 social accounts and Bitcoin wallet addresses bound. The testnet will be launched at the beginning of March for staking, and the mainnet will be launched in mid-March.
```

### Is this project associated with other projects/ecosystem stakeholders?

Yes

### If answered yes, what are the other projects/ecosystem stakeholders

```text
- Ecosystem Building: Currently, there are multiple eco projects such as AINN Swap, AINN Growth,  AINN Image, Public AI, and innovative inscription issuance protocols under construction.
```

### Describe the data being stored onto Filecoin

```text
Transaction data on the chain, as well as our AI model, and related AI Dapp application data
```

### Where was the data currently stored in this dataset sourced from

My Own Storage Infra

### If you are a data preparer. What is your location (Country/Region)

None

### Please share a sample of the data

```text
https://stake.anvm.io/bridge
https://airdrop.anvm.io/
https://airdrop.anvm.io/leaderboard
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [ ] I confirm

### What is the expected retrieval frequency for this data

Yearly

### For how long do you plan to keep this dataset stored on Filecoin

2 to 3 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America

### How will you be distributing your data to storage providers

Cloud storage (i.e. S3), HTTP or FTP server, IPFS, Shipping hard drives, Lotus built-in data transfer

### How did you find your storage providers

Slack, Filmine, Big Data Exchange, Partners

### Please list the provider IDs and location of the storage providers you will be working with.

```text
besides the previous SPs, the followings are ready: 
f03157883 based in Guangdong Mainland China,
f02826007 based in Singapore,
f01082888 based in HK
f03081958 based in Mainland China
f02827996 based in Guangdong Mainland China
```

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by NDLABS-Leo** [comment]
Hello, given that you have already completed KYB verification, we would still like to ensure the compliance of the nodes. Please disclose the nodes you plan to collaborate with in advance.

https://github.com/NDLABS-Leo/Allocator-Pathway-ND-CLOUD/issues/15#issuecomment-2210374889

**Comment by NDLABS-Leo** [comment]
@yuxlantian 
Noted. I’ve observed that the SPs you’re working with support Spark, though retrieval rates remain low. Do you have an optimization solution for improving this?
<img width="573" alt="image" src="https://github.com/user-attachments/assets/e311f1be-74ae-44f4-928a-e8a15514306f">

**Comment by NDLABS-Leo** [comment]
Alright, I suggest we start with a smaller allocation first.

**Comment by NDLABS-Leo** [comment]
2 storage providers sealed too much duplicate data - [f02826588](https://filfox.info/en/address/f02826588): 30.46%, [f02827996](https://filfox.info/en/address/f02827996): 21.75%

Note: This issue is a common problem and is due to a bug in the bot, which has existed in the community for a long time. It will not be considered in this review.

**Comment by NDLABS-Leo** [comment]
@yuxlantian 
The report shows that the overall situation is generally good.

Hello, will there be additional nodes in the next round, and will they provide enhanced retrieval services?

**Comment by NDLABS-Leo** [comment]
@yuxlantian 
Certainly, please ensure compliance with DataCap usage, and if there are any additional nodes, I will need to conduct a review of them. Thank you!

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger

**Comment by NDLABS-Leo** [comment]
checker:manualTrigger

**Comment by NDLABS-Leo** [comment]
f03081958
f01082888
Please provide an explanation of the status of these two nodes.

**Comment by NDLABS-Leo** [comment]
@NDLABS-Leo Is there a new list of partners available?

**Comment by NDLABS-Leo** [comment]
@yuxlantian 

<img width="476" alt="Image" src="https://github.com/user-attachments/assets/b504d4b0-5a9c-4701-8020-101b86245c9e" />

Looks good!

Haha, thanks for waiting. We’ve also been keeping up with the latest developments in the community recently.

Moving forward, we hope to make efforts in the following five areas:

Public Diligence
Accuracy of Storage Provider Lists
CID Duplication and Distribution
Storage and Retrievability
General Compliance Practices

**Comment by NDLABS-Leo** [comment]
checker:manualTrigger

**Comment by NDLABS-Leo** [comment]
> [@NDLABS-Leo](https://github.com/NDLABS-Leo) These two nodes are unable to continue collaboration due to technical issues with the partnered SP. Therefore, future tasks will no longer be handled by these nodes.

f01082888 ： Total Deal Size 38.88 TiB

f03081958：Total Deal Size 39.56 TiB

Today’s new round of checks has verified this response, and except for these two nodes, the retrieval rate of all other nodes is normal.

**Comment by NDLABS-Leo** [comment]
Project Closed.
